samcli/commands/check/calculation.py

```diff
@@ -7,14 +7,14 @@
 
 
 from samcli.commands._utils.resources import AWS_LAMBDA_FUNCTION
-from samcli.commands.check.resources.Graph import Graph
-from .resources.Warning import CheckWarning
+from samcli.commands.check.resources.graph import CheckGraph
+from .resources.warning import CheckWarning
 
 
 class Calculation:
```

Comment:
Just like Graph, this class have very generic name, please update it with more unique name.

Reply:
I will do the same for pricing as well